### PR TITLE
[SecurityBundle] Fix profiler dump for non-invokable security listeners

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | 4.3
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

Listeners can be any callable since 4.3.

### CODE REVIEW

File: src/Symfony/Bundle/SecurityBundle/Debug/WrappedListener.php [hunk 1]
```diff
@@ -77,7 +99,7 @@ public function getWrappedListener()
     public function getInfo(): array
     {
         if (null === $this->stub) {
-            $this->stub = self::$hasVarDumper ? new ClassStub(\get_class($this->listener)) : \get_class($this->listener);
+            $this->stub = self::$hasVarDumper ? new ClassStub($this->pretty.'()', $this->listener) : $this->pretty.'()';
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/SecurityBundle/Debug/WrappedListener.php]: Would `ClassStub::wrapCallable()` we applicable here? Would it be better?

**Review by nicolas-grekas — APPROVED** [review]
(I push-forced some tweaks)

File: src/Symfony/Bundle/SecurityBundle/Debug/WrappedListener.php [hunk 2]
```diff
@@ -76,8 +72,25 @@ public function getWrappedListener()
 
     public function getInfo(): array
     {
-        if (null === $this->stub) {
-            $this->stub = self::$hasVarDumper ? new ClassStub(\get_class($this->listener)) : \get_class($this->listener);
+        if (null !== $this->stub) {
+            // no-op
+        } elseif (self::$hasVarDumper ?? self::$hasVarDumper = class_exists(ClassStub::class)) {
+            $this->stub = ClassStub::wrapCallable($this->listener);
+        } elseif (\is_array($listener)) {
```
Comment (stof) [inline src/Symfony/Bundle/SecurityBundle/Debug/WrappedListener.php]: this looks like an undefined variable.

**Review by stof — CHANGES_REQUESTED** [review]

**Review by fabpot — APPROVED** [review]

File: src/Symfony/Bundle/SecurityBundle/Debug/WrappedListener.php (hunk 2 above)
Comment (nicolas-grekas) [reply src/Symfony/Bundle/SecurityBundle/Debug/WrappedListener.php]: oops, fixed

**Review by stof — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @chalasr.

**Pull request merged**